docs/what-is-ngrok.mdx

```diff
@@ -0,0 +1,60 @@
+# What is ngrok?
+
+---
+
+## What is ngrok?
+
+ngrok is a globally distributed reverse proxy that secures, protects and
+accelerates your applications and network services, no matter where you run
+them. You can think of ngrok as the front door to your applications.
+
+ngrok is _environment independent_ because it can deliver traffic to services
+running anywhere with no changes to your environment's networking. Run your app
+on AWS, Azure, Heroku, an on-premise Kubernetes cluster, a Raspberry Pi, and
+even your laptop. With ngrok, it all works the same.
+
+ngrok is a _unified ingress platform_ because it combines all the components to
+deliver traffic from your services to the internet into one. ngrok
+consolidates together your reverse proxy, load balancer, API gateway, firewall,
+delivery network, DDoS protection and more.
+
+## What can you do with ngrok?
+
+### Development and Testing
+
+- **Webhook Testing**: Run ngrok on your local machine to get a URL to receive
+  webhooks directly in the app you're developing. [Inspect and replay
+  requests](/agent/web-inspection-interface/) for fast development.
+- **Local Previews**: Demo a website running on your local machine to a client
+  or stakeholder without deploying to a staging stite.
+- **Mobile Backend Testing**: Test your mobile apps against a backend that
+  you're developing on your local machine.
+
+### Ingress to external networks
+
+- **APIs in customer networks**: Run the lightweight [ngrok agent](/agent/) or
+  [Kubernetes controller](/k8s/) in your customers' environemnts to securely
+  connect to APIs in their networks without complex network configuration.
+- **APIs on devices**: Run [ngrok as a service](/agent/#background-service) on
+  your devices to create secure URLs for their local APIs enabling your cloud
+  service to control and administrate them.
+- **APIs in local dev environments**: Import [ngrok as a library](/agent-sdks)
+  into your own CLI so you can create better local dev experiences for your
+  developer customers.
+
+### Production Ingress
+
+- API Gateway: Use ngrok's [HTTP modules](/http/#modules) to secure, protect,
+  accelerate and transform traffic to your production APIs.
+- Kubernetes Ingress: Run [ngrok's Ingress Controller](/k8s/) to create ingress
+  to k8s Services in any Kubernetes cluster.
+- Identity-Aware Proxy: Use ngrok's [OAuth](/http/oauth), [SAML](/http/saml/),
+  or [OpenID Connect](/http/openid-connect) modules to federate your app's
+  authentication to an identity provider.
+- Load Balancer: Use [Edges](/network-edge/edges/) to load balance traffic for
```

Comment:
- **Load Balancer:** Use [Edges](/network-edge/edges/) to load balance traffic for